dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -52,7 +52,9 @@
                           :validation-label="field.validationLabel"
                           :data-test="field.name"
                           :shouldDisableCheckboxes="true"
-                          @reportsUpdated="updateDocumentList"
+                          @field-specific-documents-updated="
+                            updateDocumentList(`${category.name}.${subcategory.name}.${field.name}`, $event)
```

Comment:
Why this addition?

Reply:
I changed the name in the base datapoint to something more fitting. And in lksg this is only an event in base data points. Thus I renamed it.

Reply:
ok